cmd/argo/commands/delete.go

```diff
@@ -40,6 +43,14 @@ func NewDeleteCommand() *cobra.Command {
 				os.Exit(1)
 			}
 
+			var workflowClientset *workflow.Clientset
+			if force {
+				clientConfig := client.GetConfig()
+				config, err := clientConfig.ClientConfig()
+				errors.CheckError(err)
+				workflowClientset = workflow.NewForConfigOrDie(config)
```

Comment:
I wonder if we can use the same "serviceAccount" variable that's being used for the deletion instead of needing to add this.